community.md

```diff
@@ -43,4 +43,8 @@ a way that makes the community thrive.
 Below is a short list of gitter channels, email listservs, and github repositories
 where you can get involved. **We always welcome participation in the Jupyter community**.
 
+## Report vulnerabilities
```

Comment:
What if we added some sub-sections here, one that was "report vulnerabilites", and another that was something like "connect with the community" or "get involved" or something?

(also friendly note that I have https://github.com/jupyter/jupyter/pull/417 open to try updating some of this stuff in the readthedocs page)

Reply:
> What if we added some sub-sections here, one that was "report vulnerabilites", and another that was something like "connect with the community" or "get involved" or something?

Can you explain what you mean by the additional sub-sections? I think I'm missing what you mean.

> (also friendly note that I have jupyter/jupyter#417 open to try updating some of this stuff in the readthedocs page)

*Taking a note to look at that PR*

I agreethis reporting information should also go in the jupyter/jupyter docs. It's on my list of places to PR tracked over in Discourse. https://discourse.jupyter.org/t/responsible-vulnerability-reporting/655/8?u=parente

Reply:
I meant have something like this:

```
## Connect online

<list of comms channels>

## Report a vulnerability

blah blah
```

Either way, it feels to me like the we'd want to ensure that the SEO of "jupyter vulnerability" or "jupyter security" surfaces whatever page we want to have that information...my guess is that many folks would google it rather than go crawling through the jupyter.org sub-pages

Reply:
@choldgraf I didn't have time to make the updates before the merge. We can tackle these in a follow-on reorg of the community page if you'd like.